echo.js

```diff
@@ -1,7 +1,11 @@
 function echo(){
-  // ...
+  for (var i = 0; i < arguments.length; i++) {
+  	console.log(arguments[i]) + "<br>";
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'console' is not defined.